sdk/kas_client.go

```diff
@@ -113,6 +113,125 @@ func (k *KASClient) unwrap(keyAccess KeyAccess, policy string) ([]byte, error) {
 	return key, nil
 }
 
+func (k *KASClient) getNanoTDFRewrapRequest(header string, kasURL string, pubKey string) (*kas.RewrapRequest, error) {
```

Comment:
Should we just update the other rewrap function to take a `RequestBody` that way it supports nano and tdf rewraps?

Reply:
You're suggesting refactoring the KAS rewrap, which is fine, but out-of-scope for this land-and-expand nanotdf support effort IMO.

@pflynn-virtru ?  Your opinion here?  If this is desirable, then can we open an issue to address this in a subsequent PR?

Reply:
Sujan did lay out a plan for optimizations that he saw; with a goal of delivering those next week.  Maybe at this to the list.
Also, the bulk rewrap endpoint (still in design phase) may affect this too.

Reply:
Created #862 to address this